### DISCUSSION TITLE
How i can add customization  in v6 and Vue3?

### DISCUSSION THREAD

**Discussion by dmitriilopotovskii** [fabricjs/fabric.js#9055] [Q&A] [answered]
in 5 version i can add customization like this [v5Vue3](https://codesandbox.io/s/fabricv5vue3-l9r2hp?file=/src/components/Canvas.vue)
in version 6 it doesn't work that way [v6Vue3](https://codesandbox.io/s/fabricv6vue3-fgw95p?file=/src/components/Canvas.vue)
How can I do that ?

**Comment by ShaMan123** [comment, marked as answer]
You really must read the breaking changes ticket

Props do not exist on the prototype any longer.

FabricObject.ownDefaults.stroke = 'blue'
